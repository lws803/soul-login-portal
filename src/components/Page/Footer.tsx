import { Center, Stack, Link, Text, Box } from '@chakra-ui/react';

export default function Footer() {
  return (
    <Center bgColor="#2C2E33">
      <Box margin="16px 0px" padding="8px 16px" width="100%">
        <Stack
          direction={['column', 'column', 'row']}
          gap={4}
          justifyContent="center"
        >
          <Link
            href="https://api.soul-network.com/docs"
            target="_blank"
            whiteSpace="nowrap"
          >
            API Docs
          </Link>
          <Link
            href="https://status.soul-network.com"
            target="_blank"
            whiteSpace="nowrap"
          >
            Status
          </Link>
          <Link
            href="https://github.com/soul-project"
            target="_blank"
            whiteSpace="nowrap"
          >
            Soul&apos;s organization Github
          </Link>
          <Link
            href="https://discord.gg/kuGNRembKZ"
            target="_blank"
            whiteSpace="nowrap"
          >
            Discord server
          </Link>
          <Link
            href="mailto:lws803@gmail.com"
            target="_blank"
            whiteSpace="nowrap"
          >
            Contact me
          </Link>
        </Stack>
        <Text
          textAlign={['left', 'left', 'center']}
          mt={[8, 8, 4]}
          fontWeight="light"
        >
          Copyright Â© {new Date().getFullYear()} Soul.
        </Text>
      </Box>
    </Center>
  );
}
