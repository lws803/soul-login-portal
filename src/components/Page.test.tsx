import { render } from '@testing-library/react';

import Page from './Page';

describe(Page, () => {
  it('renders', () => {
    const { container } = render(
      <Page errors={[]} title="TITLE">
        <p>TEST</p>
      </Page>,
    );
    expect(container).toMatchSnapshot();
  });
});
