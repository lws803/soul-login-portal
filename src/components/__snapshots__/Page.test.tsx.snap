// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders 1`] = `
<div>
  <div
    class="css-0"
  >
    <div
      class="css-1wiwcvy"
    >
      <div
        class="css-1ryq4o3"
      >
        TITLE
        <p>
          TEST
        </p>
      </div>
    </div>
    <div
      class="css-l5skqa"
    >
      <div
        class="css-18hh56p"
      >
        <div
          class="chakra-stack css-1o71ot0"
        >
          <a
            class="chakra-link css-epvm6"
            href="https://api.soul-network.com/docs"
            target="_blank"
          >
            API Docs
          </a>
          <a
            class="chakra-link css-epvm6"
            href="https://github.com/soul-project"
            target="_blank"
          >
            Soul's organization Github
          </a>
          <a
            class="chakra-link css-epvm6"
            href="mailto:lws803@gmail.com"
            target="_blank"
          >
            Contact me
          </a>
        </div>
        <p
          class="chakra-text css-1m360k3"
        >
          Copyright Â© 
          2022
           Soul.
        </p>
      </div>
    </div>
  </div>
</div>
`;
